page-changelog.php

```diff
@@ -13,7 +13,117 @@
+Packages on our Build Service: <a href="https://download.owncloud.org/download/repositories/9.1/owncloud">9.1 Release Channel</a> and <a href="https://download.owncloud.org/download/repositories/stable/owncloud/">Stable Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
+
+<a name="latest9.0"></a></li>
+<h3>Version 9.0.6 <small>Oct 26 2016</small></h3></li>
+<ul>
+<li>Core: Adjusted documentation link to issue template - <a href="https://github.com/owncloud/core/issues/26087">core/#26087</a></li>
+<li>Core: Display feedback in users page when changing password - <a href="https://github.com/owncloud/core/issues/25532">core/#25532</a></li>
+<li>Core: Fix mime type detection in hidden directories - <a href="https://github.com/owncloud/core/issues/26133">core/#26133</a></li>
+<li>Core: Change forum URL to central - <a href="https://github.com/owncloud/core/issues/25644">core/#25644</a></li>
+<li>Core: Fix share array format passed to "post_unshareFromSelf" hook - <a href="https://github.com/owncloud/core/issues/26389">core/#26389</a></li>
+<li>Core: Release mount info memory after running background jobs - <a href="https://github.com/owncloud/core/issues/26223">core/#26223</a></li>
+<li>Core: Improve users page performance by not sorting after every add - <a href="https://github.com/owncloud/core/issues/26234">core/#26234</a></li>
+<li>Core: Escape special chars in some queries - <a href="https://github.com/owncloud/core/issues/25429">core/#25429</a></li>
+<li>Files: Allow uploading empty files in the web UI - <a href="https://github.com/owncloud/core/issues/19116">core/#19116</a></li>
+<li>Sharing: Let the share owner increase permissions - <a href="https://github.com/owncloud/core/issues/25542">core/#25542</a></li>
+<li>Federation: Fix sharing with remote user names containing spaces - <a href="https://github.com/owncloud/core/issues/25955">core/#25955</a></li>
+<li>Federation: Save some memory in sync job by releasing mount info after each user - <a href="https://github.com/owncloud/core/issues/26204">core/#26204</a></li>
+<li>CardDAV: Unset photo before setting a new one - <a href="https://github.com/owncloud/core/issues/26242">core/#26242</a></li>
+<li>Updater: Fix web UI update in some environments - <a href="https://github.com/owncloud/updater/issues/378">updater/#378</a></li>
+<li>User_LDAP: Added OCC command to update group mappings - user_<a href="https://github.com/owncloud/ldap/issues/14">user_ldap/#14</a></li>
+<li>User_external: Double verify the SMB response - <a href="https://github.com/owncloud/apps/issues/2198">apps/#2198</a></li>
+<li>Antivirus: Fix background scan - files_<a href="https://github.com/owncloud/antivirus/issues/128">files_antivirus/#128</a></li>
```

Comment:
Broken link, the ``files_`` of ``files_antivirus`` is in front of the a href link